modules/serverless-blue-green-deploy.adoc

```diff
@@ -0,0 +1,112 @@
+[id="serverless-blue-green-deploy_{context}"]
+= Routing and managing traffic by using a blue-green deployment strategy
+
+You can safely reroute traffic from a live version of an app to a new version, by using a link:https://en.wikipedia.org/wiki/Blue-green_deployment[blue-green deployment strategy].
+
+.Procedure
+
+. Create and deploy an app as a Knative service.
+. Find the name of the first revision that was created when you deployed the service, by viewing the output from the following command:
++
+[source,terminal]
+----
+$ oc get configurations <service_name> -o=jsonpath='{.status.latestCreatedRevisionName}'
+----
+. Add the following YAML to the service `spec` to send inbound traffic to the revision:
++
+.Example Route
```

Comment:
Should this be `Example Service`?

Reply:
I think this heading can be removed altogether actually since I reworded this step and made it just a `spec` excerpt. Good catch, thanks